build/charts/antrea/values.yaml

```diff
@@ -165,6 +165,14 @@ antreaProxy:
   # and Antrea proxy, with both trying to bind to the same addresses, when proxyAll
   # is enabled while kube-proxy has not been removed.
   disableServiceHealthCheckServer: false
+  # -- ProxyHealthServerPort is the port for AntreaProxy health server to serve on.
+  # Defaults to 10352.
+  ProxyHealthServerPort: 10352
```

Comment:
These 2 new Helm values should not be capitalized
Also, maybe we should make the names more consistent with the exiting `disableServiceHealthCheckServer` by renaming them to `serviceHealthCheckServerPort` and `serviceHealthCheckServerBindAddress`? The proxy suffix doesn't add value since we are already under the `antreaProxy` config section.